internal/serviceconfig/serviceconfig_test.go

```diff
@@ -133,3 +135,57 @@ func TestBalancerConfigUnmarshalJSON(t *testing.T) {
 		})
 	}
 }
+
+func TestBalancerConfigMarshalJSON(t *testing.T) {
+	tests := []struct {
+		name     string
+		bc       BalancerConfig
+		wantJSON string
+	}{
+		{
+			name: "OK",
+			bc: BalancerConfig{
+				Name:   testBalancerBuilderName,
+				Config: testBalancerConfig,
+			},
+			wantJSON: fmt.Sprintf("[{%q: %v}]", testBalancerBuilderName, testBalancerConfigJSON),
+		},
+		{
+			name: "first balancer not registered",
+			bc: BalancerConfig{
+				Name:   testBalancerBuilderName,
+				Config: testBalancerConfig,
+			},
+			wantJSON: fmt.Sprintf("[{%q: %v}]", testBalancerBuilderName, testBalancerConfigJSON),
```

Comment:
The input and the output for this entry in the table seems similar to the above one. Am I missing something?

Reply:
Test case deleted.
It's copied from the unmarshal tests. It doesn't make sense for a marshal test.